src/lstm.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+
+import {concat} from './concat.js';
+import {lstmCell} from './lstm_cell.js';
+import {reshape} from './reshape.js';
+import {sizeOfShape, Tensor} from './lib/tensor.js';
+import {sigmoid} from './sigmoid.js';
+import {slice} from './slice.js';
+import {squeeze} from './squeeze.js';
```

Comment:
Now squeeze op has been removed, would you please help also remove it from this WebNN Baseline. 
You can refer to the given **squeeze method** in Spec, thanks.

```
function squeeze(builder, op) {
  return builder.reshape(op, op.shape().remove(0));
}
```

Reply:
The definition is:

```
function squeeze(input, axes) {
  if (!axes) axes = [];
  if (!axes.length)
    input.shape().forEach((item, i) => { axes.push(i); });
  shape = Array.from(input.shape());
  for (let axis in axes.sort().reverse())
    if (axis < shape.length && shape[axis] == 1)
      shape.splice(axis, 1);
  return builder.reshape(input, shape);
}
```

Reply:
OK， I will revise it next week:)